scripts/plugins/storage/postgres/upgrade/5.sql

```diff
@@ -1,17 +1,25 @@
-UPDATE foglamp.configuration SET value = '{"plugin": {"description": "OMF North Plugin", "type": "string", "default": "omf", "value": "omf"}, "source": {"description": "Source of data to be sent on the stream. May be either readings, statistics or audit.", "type": "string", "default": "audit", "value": "readings"}}'
+UPDATE foglamp.configuration SET value = jsonb_set(value, '{stream_id}', '{"type":"integer","description":"Stream ID","default":"1","value":"1"}')
+       WHERE key = 'North Readings to PI';
+UPDATE foglamp.configuration SET value = jsonb_set(value, '{source}', '{"source": {"description": "Source of data to be sent on the stream. May be either readings, statistics or audit.", "type": "string", "default": "audit", "value": "readings"}}')
```

Comment:
> "default": "audit", "value": "readings"

Using this if user deletes the source config, it will fall back to audit, which is wrong

Reply:
Fixed: 
"default": "readings", "value": "readings"
"default": "statistics", "value": "statistics"